Python/Product/PythonTools/PythonTools/PythonAnalysisClassifier.cs

```diff
@@ -99,10 +98,14 @@ private class ClassificationComparer : IEqualityComparer<AP.AnalysisClassificati
             public static readonly IEqualityComparer<AP.AnalysisClassification> Instance = new ClassificationComparer();
 
             public bool Equals(AP.AnalysisClassification x, AP.AnalysisClassification y) {
-                return x.start == y.start && x.length == y.length && x.type == y.type;
+                return x.startLine == y.startLine &&
+                    x.startColumn == y.startColumn &&
+                    x.endLine == y.endLine &&
+                    x.endColumn == y.endColumn &&
+                    x.type == y.type;
             }
 
-            public int GetHashCode(AP.AnalysisClassification obj) => obj.start;
+            public int GetHashCode(AP.AnalysisClassification obj) => obj.startLine << 8 + obj.startColumn;
```

Comment:
Why `endLine`, `endColumn` and `type` are skipped?

Reply:
Because it's just a hash code, and the start location should be unique anyway.